actioncable/lib/action_cable.rb

```diff
@@ -30,7 +30,8 @@ module ActionCable
 
   INTERNAL = {
     identifiers: {
-      ping: '_ping'.freeze
+      ping: '_ping'.freeze,
+      disconnect: '_disconnect'.freeze
```

Comment:
I'd prefer to send `disconnect` as a message type, not with an `identifier`. (I'd like to do the same for pings too, eventually. Message types were implemented after pings were introduced.)